File: WebApplication1/Controllers/HomeController.vb

<ComClassAttribute>
Public Class HomeController
    Inherits System.Web.Mvc.Controller

    Function Index() As ActionResult
        Return View()
    End Function

    Function About() As ActionResult
        ViewData("Message") = "Your application description page."

        Return View()
    End Function

    Function Contact() As ActionResult
        ViewData("Message") = "Your contact page."

        Dim message = New System.Messaging.Message

        Dim exitCode = Shell("myExe.exe")

        ChDir("")

        IO.File.WriteAllText("xyz.txt", "")

        Dim q = Microsoft.VisualBasic.MsgBoxStyle.AbortRetryIgnore

        Return View()
    End Function
End Class

File: WebApplication2/Controllers/HomeController.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using Microsoft.AspNetCore.Mvc;

namespace WebMvc.Controllers
{
    public class HomeController : Controller
    {
        public ActionResult Index()
        {
            var q = new System.Net.Http.HttpClient();

            var someUrl = "http://someadress.com/xxxxxxxx";

            return View();
        }

        public ActionResult About()
        {
            ViewBag.Message = "Your application description page.";

            return View();
        }

        public ActionResult Contact()
        {
            ViewBag.Message = "Your contact page.";

            //var message = new System.Messaging.Message();

            return View();
        }
    }
}